Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM runatlantis/atlantis-base:v3.5
 LABEL authors="Anubhav Mishra, Luke Kysow"
 
 # install terraform binaries
-ENV DEFAULT_TERRAFORM_VERSION=0.14.10
+ENV DEFAULT_TERRAFORM_VERSION=0.15.0
 
 # In the official Atlantis image we only have the latest of each Terraform version.
 RUN AVAILABLE_TERRAFORM_VERSIONS="0.8.8 0.9.11 0.10.8 0.11.14 0.12.30 0.13.6 ${DEFAULT_TERRAFORM_VERSION}" && \
```

Comment:
Shouldn't `0.14.10` be added here for folks still on the `0.14` branch?

Reply:
ah good catch, just pushed the update for this! thanks!